common/logisticspipes/proxy/side/ServerProxy.java

```diff
@@ -249,6 +251,9 @@ public boolean checkSinglePlayerOwner(String commandSenderName) {
 	@Override
 	public void openFluidSelectGui(int slotId) {}
 
+	@Override
+	public void openGuiFromItem(int guiId, EnumHand hand) {}
```

Comment:
Call this something more descriptive. `openGuideBookGui`?

Reply:
Very well it ended up not being id based so it doesn't make sense to have a general name